src/DataBase/EntryDbUpdater.cs

```diff
@@ -103,32 +103,32 @@ public static bool RemoveEntryList(EntryList entryList)
         }
 
         //As I understand, all the entry lists that were passed to this Method will now have amended Id's and user Id's and are ready to be added locally 
-        public static bool AddEntryList(EntryList entryList, int userId)
+        public static bool AddRange(IEntryEnumerable entryList, int userId)
         {
             var db = new DatabaseContext();
 
             if (entryList.EntryType == EntryType.Income)
             {
-                var databaseEntryList = new List<Incomes>();
+                var dbEntryList = new List<IncomeModel>();
                 foreach (var entry in entryList)
                 {
-                    var databaseEntry = new Incomes(entry, userId);
-                    databaseEntryList.Add(databaseEntry);
-                    entry.Id = databaseEntry.Id;
+                    var dbEntry = new IncomeModel(entry, userId);
+                    dbEntryList.Add(dbEntry);
+                    entry.Id = dbEntry.Id;
                     entry.UserId = userId;
                 }
-                db.AddRange(databaseEntryList);
+                db.AddRange(dbEntryList);
             }
             else
             {
-                var databaseEntryList = new List<Expenses>();
+                var dbEntryList = new List<ExpenseModel>();
                 foreach (var entry in entryList)
                 {
-                    var databaseEntry = new Expenses(entry, userId);
-                    databaseEntryList.Add(databaseEntry);
-                    entry.Id = databaseEntry.Id;
+                    var dbEntry = new ExpenseModel(entry, userId);
+                    dbEntryList.Add(dbEntry);
+                    entry.Id = dbEntry.Id;
                 }
-                db.AddRange(databaseEntryList);
+                db.AddRange(dbEntryList);
             }
 
             db.SaveChanges();
```

Comment:
I renamed the models, because (at least to me) having classes named Goal and Goals was a bit unclear and I would get confused a lot :D I also deleted old migrations which weren't necessary